packages/twenty-server/src/engine/core-modules/billing/services/billing-portal.workspace-service.ts

```diff
@@ -69,14 +69,18 @@ export class BillingPortalWorkspaceService {
     workspaceId: string,
     returnUrlPath?: string,
   ) {
-    const currentSubscriptionItem =
+    const currentSubscription =
       await this.billingSubscriptionService.getCurrentBillingSubscriptionOrThrow(
         {
           workspaceId,
         },
       );
 
-    const stripeCustomerId = currentSubscriptionItem.stripeCustomerId;
+    if (!currentSubscription) {
+      return;
```

Comment:
I think we should throw here (and rename computeBillingPortalSessionURLOrThrow() )